models/reports_tables/reports_by_location.sql

```diff
@@ -0,0 +1,19 @@
+{{
+    config(
+        materialized = 'incremental',
+    )
+}}
+
+SELECT
+    couchdb.doc ->> '_id'::text AS uuid,
+    couchdb.doc ->> 'form'::text AS form,
+    CAST (coalesce(couchdb.doc #>> '{geolocation,longitude}', '36.826275') AS NUMERIC) AS longitude,
+    CAST (coalesce(couchdb.doc #>> '{geolocation,latitude}', '-1.1792353') AS NUMERIC) AS latitude,
+    couchdb.doc ->> 'reported_date'::text AS reported_date,
+    couchdb.doc #>> '{contact,_id}'::text[] AS contact_uuid
+FROM {{ ref("couchdb") }}
+WHERE (couchdb.doc ->> 'type'::text) = 'data_record'
+
+{% if is_incremental() %}
+    AND COALESCE("@timestamp" > (SELECT MAX("@timestamp") FROM {{ this }}), True)
```

Comment:
Wrong use of aggregate function according to DBT

```sh
15:49:34    aggregate functions are not allowed in WHERE
15:49:34    LINE 26:     AND COALESCE("@timestamp" > (SELECT MAX("@timestamp") FR...
```

Reply:
I'm getting the same error as @samuelimoisili